src/app/pages/dashboard/components/widget-sys-info/widget-sys-info.component.ts

```diff
@@ -106,7 +106,7 @@ export class WidgetSysInfoComponent extends WidgetComponent implements OnInit, O
         this.ha_status = haStatus.status;
       });
     } else {
-      this.store$.pipe(waitForSystemInfo, untilDestroyed(this)).subscribe(
+      this.ws.call('system.info').pipe(untilDestroyed(this)).subscribe(
```

Comment:
Instead of making a new call, I think it would be better if we store a timestamp of when the original call came back with a response. And have another selector that adds the difference to the nas time stored in state.
Something like
`this.store$.select(addedNasTime())` or something.